### ISSUE TITLE
onboarding feedback

### ISSUE THREAD

**Issue by atherdon** [GroceriStar/creative#7]
As you might now, we also tried to create some app screens with another 
designer. And in case when i can request changes, i think it's important to compare work.

As i think that Now UI is a good started to such kind of app - i think it can be discussed 
what other pages can be created(it's like a brainstorm, when we sit, review all assets that we have and discuss ideas)

1) i moved your files into our creative repository. just to let you know

We also have a dashboard that what created by separated designer too, it can also give some details.

I also add you into our AL project, i hope it'll generate an additional attention to both sides.

in short - i like your onboarding screens - i like the feeling that they give. it's awesome.

i also curious to hear your opinion about having an autocomplete at input field(screen where user picks "categroeis")

this autocomplete can have custom design or be usual iOS/Android app.
please advice what do you think.

Does onboarding run only on registration? What screens you think should be added in order to make our app more cool?
or usable for users?

What do you think about this case:
User don't have a lot of knowlege in terms, related to terms that we're using.
should we add some help section or hint in order to prevent it?
